lib/rules/prefer-async-await.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @fileoverview Prefer async/await over .then()
+ * @author Elad Shahar
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Helpers
+//------------------------------------------------------------------------------
+
+/**
+ * Determine if a node is calling a `then` property
+ * @private
+ * @param {ASTNode} node - node to test
+ * @returns {boolean} True if `node` is a `then` property call
+ */
+function isThenCall(node) {
+    return node.property.type === "Identifier" &&
+           node.property.name === "then";
+}
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+    meta: {
+        docs: {
+            description: "require usage of async/await over .then()",
+            category: "Best Practices",
+            recommended: false,
+            url: "https://eslint.org/docs/rules/prefer-async-await"
+        },
+        schema: [],
+        messages: {
+            unexpected: "Prefer using async/await over .then()"
```

Comment:
Can you add a period to the end of this message? (It's a minor thing, but the we're trying to keep report messages consistent across all rules. )

> `Prefer using async/await over .then().`